recipes/openff-nagl/meta.yaml

```
@@ -0,0 +1,52 @@
+{% set name = "openff-nagl" %}
+{% set version = "0.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/openforcefield/openff-nagl/archive/v{{ version }}.tar.gz
+  sha256: d9e9c60a5103ae5f091cfb1378e4ac3526d3edde952115c33f28005b0928e8b3
+
+build:
+  skip: true  # [win]
+  skip: true  # [py<38]
```

Since this is pure Python, we can make it noarch:
```suggestion
  noarch: python
```
These are by default only built on Linux and we can ignore the Windows failures due to missing dependencies (which will keep the package from installing on Windows).